addon/components/o-s-s/attribute-text.ts

```diff
@@ -0,0 +1,28 @@
+import { assert } from '@ember/debug';
+import { isBlank } from '@ember/utils';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+interface OSSAttributeTextArgs {
+  label: string;
+  value?: string;
+  copyable?: boolean;
+  tooltip?: string;
+}
+
+export default class OSSAttributeText extends Component<OSSAttributeTextArgs> {
+  @tracked displayCopyBtn = false;
+  @tracked copyable = this.args.copyable ?? true;
+  constructor(owner: unknown, args: OSSAttributeTextArgs) {
+    super(owner, args);
+    assert(`[component][OSS::Attribute::Text] label is required `, this.args.label);
+  }
+
+  get value(): string {
+    return isBlank(this.args.value) ? '-' : this.args.value ?? '-';
```

Comment:
i don't think the `?? '—'` at the end is necessary as `isBlank(undefined)` returns true

Reply:
```suggestion
    return isBlank(this.args.value) ? '-' : this.args.value;
```

Reply:
TS doesn't like it when I don't check for it @phndiaye  😢